src/homepage/pages/Home.vue

```diff
@@ -22,116 +22,111 @@
   </section>
 </template>
 <script>
-export default {
-  name: 'homepage',
-  mounted() {
-    setTimeout(function() {
-      particlesJS.load('meteor', 'static/json/particles.json', () => {
-      });
-    }, 2000);
-  }
-};
+  require('particles.js/particles.js');
```

Comment:
I tried to add `'use strict';` in `Home.vue` but there is still no error.  
And there is a question why call `setTimeout`